docs/graphql/manual/remote-schemas/index.rst

```diff
@@ -80,9 +80,10 @@ You need to enter the following information:
 
   .. note::
 
-    During **local development** using Docker and a localhost server, ensure the Hasura Docker container can reach
-    the server endpoint on the host, i.e. use ``host.docker.internal`` on Mac or ``docker.for.win.localhost`` on
-    Windows.
+    During **local development** with the remote schema server on localhost, ensure that the Hasura Docker container can reach
+    the server endpoint on the host. A common way to do this is to use the base domain ``host.docker.internal`` (Mac) or ``docker.for.win.localhost`` (Windows) instead of ``127.0.0.1``. 
```

Comment:
```suggestion
    the server endpoint on the host. A common way to do this is to use the domain ``host.docker.internal`` on Docker for Mac and Windows or the Docker bridge IP on Linux (typically ``172.17.0.1`` - the IP address of ``docker0`` interface. use ``ip addr show docker0``) instead of ``127.0.0.1``. 
```

Reply:
`docker.for.win.localhost` is deprecated and the recommended way is to use `host.docker.internal` on both Mac and Windows. see https://docs.docker.com/docker-for-windows/networking/

Reply:
@marionschleifer We should probably add this to other part of the docs too.